recipe/patches/0005-Remove-all-dependencies-from-setup.py.patch

```diff
@@ -1,34 +1,27 @@
-From cef3150908795045cc870326b598ac3778a4efc9 Mon Sep 17 00:00:00 2001
+From dc32b341c09175ae0e5bf77186d1e1279c533ef8 Mon Sep 17 00:00:00 2001
 From: Kai Fricke <kai@anyscale.com>
 Date: Fri, 3 Dec 2021 10:55:23 -0800
-Subject: [PATCH 3/5] Remove all dependencies from setup.py
+Subject: [PATCH 5/5] Remove all dependencies from setup.py
 
 Signed-off-by: Jiajun Yao <jeromeyjj@gmail.com>
 Signed-off-by: Matti Picus <matti.picus@gmail.com>
 ---
- python/setup.py | 107 +++++-------------------------------------------
- 1 file changed, 11 insertions(+), 96 deletions(-)
+ python/setup.py | 98 ++++---------------------------------------------
+ 1 file changed, 8 insertions(+), 90 deletions(-)
 
 diff --git a/python/setup.py b/python/setup.py
-index 6c60acbc5..aa63c8d7d 100644
+index affa582097..a26f61f8fb 100644
 --- a/python/setup.py
 +++ b/python/setup.py
-@@ -209,82 +209,21 @@ ray_files += [
+@@ -226,82 +226,18 @@ ray_files += [
  # also update the matching section of requirements/requirements.txt
  # in this directory
  if setup_spec.type == SetupType.RAY:
--    if sys.version_info >= (3, 7):
--        pandas_dep = "pandas >= 1.3"
--        numpy_dep = "numpy >= 1.20"
--    else:
--        # Pandas dropped python 3.6 support in 1.2.
--        pandas_dep = "pandas >= 1.0.5"
--        # NumPy dropped python 3.6 support in 1.20.
--        numpy_dep = "numpy >= 1.19"
--    if sys.version_info >= (3, 7) and sys.platform != "win32":
+-    pandas_dep = "pandas >= 1.3"
```

Comment:
does this need to be done if we're using `--no-deps --no-build-isolation`?

Reply:
We still need this patch yes. It allows `pip check` to work, or at least produce less errors.